cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/PreviewDataSubscriberService.java

```diff
@@ -110,13 +110,13 @@ protected TransactionRunner getTransactionRunner() {
   @Nullable
   @Override
   protected String loadMessageId(StructuredTableContext context) throws Exception {
-    AppMetadataStore appMetadataStore = AppMetadataStore.create(context);
+    AppMetadataStore appMetadataStore = AppMetadataStore.create(context, metricsCollectionService);
```

Comment:
Will this also emit metrics for preview deployment?

Reply:
I think we should remove this